packages/gitbook/src/components/Header/Dropdown.tsx

```diff
@@ -18,8 +18,10 @@ export function Dropdown<E extends HTMLElement>(props: {
     button: (buttonProps: DropdownButtonProps<E>) => React.ReactNode;
     /** Content of the dropdown */
     children: React.ReactNode;
+    /** Custom styles */
+    className?: string;
```

Comment:
We use the name `styles?: ClassValue` in this repo (I know the name is terrible).

Maybe we should start switching to `className`, but you can do:

```suggestion
    className?: ClassValue;
```